packages/database/README.md

```diff
@@ -0,0 +1,33 @@
+# Datahub database
+
+This package is the database connector needed to read and write Datahub data. To connect to the database via the researcher app, ensure the environment variable DATABASE_URL is set in `apps/researcher/.env.local`. If you want to connect to the database outside the app to run scripts, add the DATABASE_URL to `packages/database/.env.local`.
```

Comment:
1. `researcher app` -> `Research App`?
2. Add backticks to `DATABASE_URL` (to be consistent with the notation in the main README.md)?